src/main/resources/META-INF/spring/applicationContext-security.xml

```diff
@@ -2,16 +2,19 @@
 <beans:beans xmlns="http://www.springframework.org/schema/security" 
     xmlns:beans="http://www.springframework.org/schema/beans" 
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
-    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-3.1.xsd
-        http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-3.2.xsd">
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+        http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-5.6.xsd">
 
 	<!-- HTTP security configurations -->
 	<http auto-config="true" use-expressions="true">
 		<form-login login-processing-url="/resources/j_spring_security_check"
-			login-page="/login" authentication-success-handler-ref="postSuccessAuthHandler"
+			login-page="/login" 
+			username-parameter="j_username"
+			password-parameter="j_password" 
+			authentication-success-handler-ref="postSuccessAuthHandler"
 			authentication-failure-url="/login?login_error=t" />
 		<logout logout-url="/resources/j_spring_security_logout" />
-
+		<csrf disabled="true"/>
```

Comment:
`csrf` Is enabled by default. I disabled it because our calls between ACAS and roo do not support passing this back and forth.